Test failure in `test_qir_submission_mz_to_reg_qa`

See e.g. https://github.com/CQCL/pytket-quantinuum/actions/runs/9361053687/job/25767404739

```
 FAILED integration/backend_test.py::test_qir_submission_mz_to_reg_qa[H1-1SC] - assert 256 == 128
 +  where 256 = len([d[63], d[62], d[61], d[60], d[59], d[58], ...])
 +    where [d[63], d[62], d[61], d[60], d[59], d[58], ...] = <bound method BackendResult.get_bitlist of BackendResult(q_bits={},c_bits={d[63]: 0, d[62]: 1, d[61]: 2, d[60]: 3, d[5... 0 0]\n [0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0]],state=None,unitary=None,density_matrix=None)>()
 +      where <bound method BackendResult.get_bitlist of BackendResult(q_bits={},c_bits={d[63]: 0, d[62]: 1, d[61]: 2, d[60]: 3, d[5... 0 0]\n [0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0]],state=None,unitary=None,density_matrix=None)> = BackendResult(q_bits={},c_bits={d[63]: 0, d[62]: 1, d[61]: 2, d[60]: 3, d[59]: 4, d[58]: 5, d[57]: 6, d[56]: 7, d[55]:...0 0 0]\n [0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0]],state=None,unitary=None,density_matrix=None).get_bitlist
```

--- a/tests/integration/backend_test.py
+++ b/tests/integration/backend_test.py
@@ -1023,7 +1023,6 @@ def test_qir_submission_mz_to_reg(
     assert len(r.get_bitlist()) == 128
 
 
-@pytest.mark.xfail(reason="https://github.com/CQCL/pytket-quantinuum/issues/443")
 @pytest.mark.skipif(skip_remote_tests, reason=REASON)
 @pytest.mark.parametrize(
     "authenticated_quum_backend_qa", [{"device_name": "H1-1SC"}], indirect=True

